openff/toolkit/tests/test_links.py

```diff
@@ -44,7 +44,7 @@ def test_readme_links(readme_link):
     # Try to connect 5 times, keeping track of exceptions so useful feedback can be provided.
     success = False
     exception = None
-    for retry in range(5):
+    for retry in range(5):  # noqa: B007
```

Comment:
In this case I think it's more readable to keep the loop variable in; if I was reading `for _ in range(5):` I'd have to do a small amount of work to figure out that it's just doing 5 retries.

I don't feel strongly about this and am fine properly "fixing" it

Reply:
Totally agree with this reasoning and I'm in favor of keeping it readable. Good call.